ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -343,7 +346,10 @@ public String toString(final FormatOptions formatOptions) {
    * @return the LogicalSchema created, with the corresponding pseudo and key columns included
    */
   private LogicalSchema rebuildWithPseudoAndKeyColsInValue(
-      final boolean windowedKey, final int pseudoColumnVersion) {
+      final boolean windowedKey,
+      final int pseudoColumnVersion,
+      final boolean isPullOrScalablePushQuery
```

Comment:
nit: 
```suggestion
      final boolean forPullOrScalablePushQuery
```
to match the name in `withPseudoAndKeyColsInValue()` above.